### ISSUE TITLE
Segmentation fault errors in 2.8.1

### ISSUE THREAD

**Issue by alexmnyc** [helm/helm#3546]
When trying to template out yaml (from wget -q https://storage.googleapis.com/kubernetes-helm/helm-v2.8.1-linux-amd64.tar.gz)

7289 Segmentation fault      
```helm template $WORKSPACE/deployment --set POOL=$POOL --set ENV=$ENV --set REGION=$REGION --set NAMESPACE=$NAMESPACE --set VERSION=$VERSION > $YAML```

I am getting Segmentation faults on ```Linux 2ab53e64d41a 4.9.70-25.242.amzn1.x86_64 #1 SMP Wed Jan 3 05:36:22 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux```

```cat /proc/meminfo
MemTotal:       30860384 kB
MemFree:        26673688 kB
MemAvailable:   29721604 kB
Buffers:          125448 kB
Cached:          2874228 kB
SwapCached:            0 kB
Active:          1459472 kB
Inactive:        1992940 kB
Active(anon):     458604 kB
Inactive(anon):       44 kB
Active(file):    1000868 kB
Inactive(file):  1992896 kB
Unevictable:       17572 kB
Mlocked:           17572 kB
SwapTotal:             0 kB
SwapFree:              0 kB
Dirty:                88 kB
Writeback:             0 kB
AnonPages:        446520 kB
Mapped:           108052 kB
Shmem:                88 kB
Slab:             589808 kB
SReclaimable:     501564 kB
SUnreclaim:        88244 kB
KernelStack:       10432 kB
PageTables:         6816 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:    15430192 kB
Committed_AS:    2521104 kB
VmallocTotal:   34359738367 kB
VmallocUsed:           0 kB
VmallocChunk:          0 kB
AnonHugePages:         0 kB
ShmemHugePages:        0 kB
ShmemPmdMapped:        0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
DirectMap4k:       12288 kB
DirectMap2M:     2084864 kB
DirectMap1G:    30408704 kB
```

**Comment by alexmnyc** [comment]
That is more prominent when running the process in parallel for 1-8 files

**Comment by alexmnyc** [comment]
sig-testing, kubernetes/test-infra and/or @fejta.
/lifecycle

**Comment by bacongobbler** [comment]
Closing due to inactivity. Please open a new ticket against Helm 3 if this issue persists. Thanks!
